config/initializers/vips.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+Vips.block_untrusted(true) if Vips.at_least_libvips?(8, 13)
```

Comment:
Maybe instead or in addition to that, it could make sense to only enable loaders we know we will use?

Something like:
```ruby
Vips.block('VipsForeign', true)
%w(VipsForeignLoadJpeg VipsForeignLoadPng VipsForeignLoadWebp VipsForeignLoadHeif VipsForeignSavePng VipsForeignSaveJpeg VipsForeignSaveWebp).each do |operation|
  Vips.block(operation, false)
end
```

Reply:
Unfortunately, `Vips.block` itself is only available for >= 8.13 as well, but I think we should really require this, given that distributions of `libvips` can include unsafe handlers for lots of different file formats.

Reply:
Solving this conversation with a hard requirement on Vips 8.13+